infra/elasticsearch/iam.tf

```diff
@@ -0,0 +1,43 @@
+data "aws_iam_policy_document" "analytics_logging_role_policy" {
+  "statement" {
```

Comment:
Are the quotes (`"`) alright here? I didn't realise you could use quotes here. As long as it works it doesn't matter really.

Reply:
good catch although it does work it.